dev_guide/deployments.adoc

```diff
@@ -273,6 +273,14 @@ image stream changes and the new tag value differs from the current image
 specified in the deployment configuration's `helloworld` container, a new
 deployment is created using the new tag value for the `helloworld` container.
 
+The following table explains the possible combinations between ConfigChange
+and ImageChange triggers.
+
+  Trigger                  | true                                      | false
+  -----------------------  | ----------------------------------------- | ------------------------------------------------------
+  ConfigChange             | Deployments are started automatically     | Deployments need to be started manually
+  ImageChange (automatic)  | Images are always resolved automatically  | Only the initial image will be resolved in the template
```

Comment:
@kargakis: my scenario was a DeploymentConfig using an image external to
openshift (think docker.io/nginx or whatever) with a ConfigChangeTrigger and
(for a reason that's not relevant to the discussion) an ImageChangeTrigger.
The problem is the deployment is not started automatically when the DC is
created, since no imagestream exists.

I don't think this case has any practical application, as it is fairly
artificial.  Still, I was surprised adding a ICT to a DC would prevent the CCT
from working.

I believe the relevant parts of the code are:

https://github.com/openshift/origin/blob/6e83535e99fad9420ea8979e97701700905ba9ed/pkg/deploy/controller/configchange/controller.go#L50
https://github.com/openshift/origin/blob/6e83535e99fad9420ea8979e97701700905ba9ed/pkg/deploy/controller/configchange/controller.go#L92